includes/agentic-commerce/class-wc-stripe-agentic-commerce-integration.php

```
@@ -243,6 +265,68 @@ public function cancel_pending_full_resync(): void {
+
+		$force_disable_checkout = static function () {
+			return true;
+		};
+		add_filter( 'woocommerce_agentic_commerce_disable_checkout', $force_disable_checkout );
+
+		try {
+			$this->run_feed_sync( true );
+		} finally {
+			remove_filter( 'woocommerce_agentic_commerce_disable_checkout', $force_disable_checkout );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method push_final_checkout_disabled_feed is not named in camelCase. (undefined)

(CamelCaseMethodName)